src/LogDocumentLinkProvider.ts

```diff
@@ -20,10 +20,13 @@ export class LogDocumentLinkProvider implements vscode.DocumentLinkProvider {
         this.launchConfig = launchConfig;
         this.fileMaps = {};
 
-        let sourceRootDir = launchConfig.debugRootDir ? launchConfig.debugRootDir : launchConfig.rootDir;
-
+        let sourceRootDir = launchConfig.sourceDirs ? launchConfig.sourceDirs : launchConfig.rootDir;
```

Comment:
Same as above, verify, but I believe you need to wrap rootDir in array brackets so that `sourceRootDir` is always an array.